digital_twins/o5g/README.md

```diff
@@ -0,0 +1,48 @@
+## Overview
+This example is an implementation of the the paper Digital Twin for Rescue Missions--a Case Study by Leucker et al.: https://ceur-ws.org/Vol-3507/paper4.pdf
+
+## Lifecycle
+
+### Create
+
+Before running the create script please download the `tessla-telegraf-connector.zip` from https://git.tessla.io/telegraf/tessla-telegraf-connector and extract its contents to `tools/tessla/tessla-telegraf-connector`.  
+Also make sure all the files of this exampla are present at the correct locations as specified at the top of `lifecycle/create`.
```

Comment:
Perhaps it would be better to have the list of files here in the README instead of as comments in the script